tests/e2e/registry/test_registry_pull_and_push_images_to_registry.py

```diff
@@ -43,20 +43,13 @@ def test_registry_pull_and_push_images(self):
 
         """
 
-        # Image pull and push to registry
-        image_pull(image_url=IMAGE_URL)
-        self.image_path = image_push(
-            image_url=IMAGE_URL, namespace=OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
+        # Pull and push images to registries
+        log.info("Pull and push images to registries")
+        image_pull_and_push(
+            project_name=self.project_name, template='eap-cd-basic-s2i',
+            image='registry.redhat.io/jboss-eap-7-tech-preview/eap-cd-openshift-rhel8:latest',
+            pattern='eap-app'
         )
 
```

Comment:
How are we confirming that the image is present in the registry?

Reply:
added a function to check the image exists on registries path